### PR TITLE
Gruntfile not finding source sass files

### PR DESCRIPTION
**Pull request by brown-a2**
@tatyree when you get a chance and you have a look at.

### CODE REVIEW

**Comment by IruneItoiz** [comment]
Does this work if you build the image from scratch using this new gruntfile?

**Comment by brown-a2** [comment]
Not sure, I was wondering that as well. one for Todd

File: docker/Gruntfile.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module.exports = function(grunt) {
   "use strict";
 
-  var srcDir = grunt.option('source-path') || 'web/app/themes';
+  var srcDir = grunt.option('source-path') || 'bedrock/web/app/themes';
```
Comment (tatyree) [inline docker/Gruntfile.js]: Which sass file is it failing to build? At which point of the build process?

**Comment by tatyree** [comment]
@brown-a2 can you post a brief outline of the steps to take to reproduce the build failure including the error message and the particular file that it's failing to build.  If it's more than a couple of steps, can you open it as an issue on the project and link to this PR, please?

**Comment by brown-a2** [comment]
Running grunt in the root of the `wp_local_dev` container (after it has been built) returns the following error below. 

```Running "sass:dev" (sass) task
>> Source file "web/app/themes/mojintranet/assets/css/src/style.scss" not found.
>> Source file "web/app/themes/mojintranet/assets/css/src/ie.scss" not found.
>> Source file "web/app/themes/mojintranet/assets/css/src/fonts.scss" not found.```

The error is solved by modifying the source path in the Gruntfile.js (line 4) to include the newly created /bedrock/ folder but this then breaks the container on build.

**Comment by IruneItoiz** [comment]
This could be fixed if we move the grunt files inside the theme, as then it would be self contained?

**Pull request closed without merging**